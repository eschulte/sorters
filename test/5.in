804 683 473 892 689 422 365 896 871 384 80 101 817 419 460 419 837 627 681 92 566 935 69 768 727 442 252 878 948 987 158 973 977 461 747 715 108 658 185 908 2 792 376 277 383 402 356 724 287 20 112 424 624 888 791 447 831 961 94 540 298 655 908 350 75 90 864 627 82 82 554 891 674 759 609 630 965 469 707 346 727 629 829 689 70 219 788 425 318 935 113 280 700 411
