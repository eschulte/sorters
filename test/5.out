2 20 69 70 75 80 82 82 90 92 94 101 108 112 113 158 185 219 252 277 280 287 298 318 346 350 356 365 376 383 384 402 411 419 419 422 424 425 442 447 460 461 469 473 540 554 566 609 624 627 627 629 630 655 658 674 681 683 689 689 700 707 715 724 727 727 747 759 768 788 791 792 804 817 829 831 837 864 871 878 888 891 892 896 908 908 935 935 948 961 965 973 977 987
